### PR TITLE
Template designer improvements

### LINKED ISSUE #208: Template revision/cloning/JSON import
Given that it does not seem a good idea to change if there are DMP connected to that (see #207), it would be nice to be able to create a new template based on a previous one (like a new revision).
Current templates can be exported as JSON file, but there is no import function for those files in the admin.

### PR DESCRIPTION
**Pull request by hmpf**
Closes #208

This will allow a Template Designer to create copies (new versions or private copies) of templates they already have access to.

Template import for Template Designers will be a separate PR.

### CODE REVIEW

**Comment by hmpf** [comment]
Currently, the copy isn't hooked up properly to the original import.

**Comment by hmpf** [comment]
I'm considering splitting this into two PR's, one with copy/clone and one with import, because it would be smart to have better protection of published templates *first*, before willy-nilly import of templates is ok.

File: src/easydmp/auth/utils.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    'generate_default_permission_strings',
+    'set_permissions',
+]
+
+
+def generate_default_permission_strings(model_name):
+    perms = []
+    for perm in ('add', 'change', 'delete', 'view'):
+        perms.append('{}_{}'.format(perm, model_name))
+    return perms
```
Comment (vbhavdal) [inline src/easydmp/auth/utils.py:16]: Access control code without coverage @hmpf

Comment (hmpf) [reply src/easydmp/auth/utils.py:16]: I'll add tests.

**Comment by hmpf** [comment]
It is possible to generate cloning-information for a direct clone of an imported template after the fact, so this PR is ready for review/merge.

**Review by vbhavdal — APPROVED** [review]

**Pull request merged**